opentelemetry-api/tests/patcher/test_patcher.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# type: ignore
+
+from unittest import TestCase
+
+from opentelemetry.patcher import BasePatcher
+
+
+class TestSampler(TestCase):
```

Comment:
Is this class name a copy & paste error?

Reply:
I'm now a bit surprised I copied that file to begin with :sweat_smile:.